### PR TITLE
Implement create message API in message form

### PR DESCRIPTION
**Pull request by yovic4**
I tried to implement create message API for the message form.

Few things to note:
- I add axios for making the request
- I change the .env format for server_url so that it can be used at browser side too

### CODE REVIEW

File: packages/frontend/interfaces/message.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export interface MessageInterface {
+  creator: string
+  // Currently doesn't exist in backend
+  // avatar_url: string
```
Comment (RedMaps) [inline packages/frontend/interfaces/message.ts]: This has been removed since we wont be using it

File: packages/frontend/.env.example [hunk 2]
```diff
@@ -1,2 +1,2 @@
 PORT=3000
-SERVER_URL="http://localhost:5000"
+NEXT_PUBLIC_SERVER_URL="http://localhost:5000/public/v1"
```
Comment (RedMaps) [inline packages/frontend/.env.example]: I'd call this API_URL or something for clarity and shortness

File: packages/frontend/api/message.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { MessageInterface } from '../interfaces/message'
+import axios from 'axios'
+
+const messageInstance = axios.create({
+  baseURL: `${process.env.NEXT_PUBLIC_SERVER_URL}/message`,
+})
+
+export async function createMessage(data: MessageInterface) {
+  await messageInstance.post('', data)
```
Comment (RedMaps) [inline packages/frontend/api/message.ts]: I think you should move the specific path to the specific function.
```suggestion
  baseURL: process.env.NEXT_PUBLIC_SERVER_URL,
})

export async function createMessage(data: MessageInterface) {
  await messageInstance.post('/message', data)
```

File: packages/frontend/components/Forms/FormMessage.tsx [hunk 4]
```diff
@@ -25,22 +22,23 @@ interface errorType {
...
+    const data = formDataToObject(formData) as MessageInterface
 
     data.creator = data.creator.trim()
     if (!data.creator) {
       return setErrors({ creator: "This field can't be empty" })
     }
 
-    data.avatar_url = data.avatar_url.trim()
+    // Currently doesn't exist in backend
+    // data.avatar_url = data.avatar_url.trim()
```
Comment (RedMaps) [inline packages/frontend/components/Forms/FormMessage.tsx]: This has been removed since we wont be using it.

**Review by RedMaps — CHANGES_REQUESTED** [review]

File: packages/frontend/interfaces/message.ts (hunk 1 above)
Comment (yovic4) [reply packages/frontend/interfaces/message.ts]: So, I can remove the input from the message form too right?

File: packages/frontend/.env.example (hunk 2 above)
Comment (yovic4) [reply packages/frontend/.env.example]: ok changing it to `NEXT_PUBLIC_API_URL`

File: packages/frontend/interfaces/message.ts (hunk 1 above)
Comment (yovic4) [reply packages/frontend/interfaces/message.ts]: image
Done, the form become like this

File: packages/frontend/.env.example (hunk 2 above)
Comment (yovic4) [reply packages/frontend/.env.example]: done

File: packages/frontend/components/Forms/FormMessage.tsx (hunk 4 above)
Comment (yovic4) [reply packages/frontend/components/Forms/FormMessage.tsx]: done

**Review by azuraga — APPROVED** [review]

**Review by RedMaps — APPROVED** [review]
Sorry for taking so long!

**Pull request merged**